docs/docs/libs/interfaces.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 1
+---
+
+# Purpose
```

Comment:
Having the section named purpose makes me think that its about like philosophy. (Like the button for next says Purpose). Could we change it to Interfaces or something?

![Screenshot from 2021-05-03 20-01-53](https://user-images.githubusercontent.com/18239203/116947352-82fb5980-ac4a-11eb-94c6-0507abb76e5b.png)

Reply:
Or like Library

Reply:
Oh yes, I meant that to be Interfaces, I must have missed that